packages/application-extension/src/index.tsx

```diff
@@ -315,10 +336,13 @@ const mainCommands: JupyterFrontEndPlugin<void> = {
       commands.addCommand(CommandIDs.toggleRightArea, {
         label: trans.__('Show Right Sidebar'),
         execute: () => {
+          let ariaLiveRegion = document.getElementById('commands-aria-live');
           if (labShell.rightCollapsed) {
             labShell.expandRight();
+            ariaLiveRegion!.append(trans.__('Right Sidebar Closed'));
```

Comment:
I do not follow why it says "Right Sidebar Closed". It was closed but it is getting opened, right? I just don't feel that the content of these hints is exactly right.